src/west/commands/command.py

```diff
@@ -170,6 +170,10 @@ def _ext_specs(project):
             'west-commands file {} escapes project path {}'.
             format(project.west_commands, project.path))
 
+    # Project may not be cloned yet.
+    if not os.path.exists(spec_file):
```

Comment:
Would this be better handled in the `with open()` below catching the exception?

Reply:
Also, should we verify that the commands themselves exist, beyond the spec file?

Reply:
can be done later.
currently working on additional test / commit to ensure we don'tr allow duplicates, e.g proj1 defining `ext_cmd_a` as well as proj2 also defining the same command.

but not as important as this PR

Reply:
> Would this be better handled in the `with open()` below catching the exception?

personally I prefer testing before trying.
but that's just my preference.

Reply:
> Also, should we verify that the commands themselves exist, beyond the spec file?

This should raise a BadExternalCommand. I'd prefer to handle that with an extra exception handler in main.